src/Std/Data/DHashMap/Internal/List/Associative.lean

```diff
@@ -2968,15 +2968,15 @@ theorem getValueCast!_alterKey {k k' : α} [Inhabited (β k')] {f : Option (β k
     (l : List ((a : α) × β a)) (hl : DistinctKeys l) : getValueCast! k' (alterKey k f l) =
       if heq : k == k' then
         haveI : Inhabited (β k) := ⟨cast (congrArg β <| eq_of_beq heq).symm default⟩
```

Comment:
This line is no longer needed, yes?

Reply:
Oh, I see that I missed the actual point of the change... Very nice.